Implemented backend subscriber manager

svparijs opened the pull request:
Added basic implementation for CRUD of subscribers via the backend.

svparijs committed:
[TASK] Added Backend Module for Subscribers

Added Dutch translations to frontend.
Fixed dimensions exception.

svparijs committed:
[FIX] Changed policy method & added Layouts

svparijs committed:
[TASK] Add missing partial

dimaip commented on Classes/Psmb/Newsletter/Controller/Module/SubscriberController.php:
```diff
@@ -0,0 +1,118 @@
+
+/**
+ * Class SubscriberController
+ * @package RFY\BKWI\Controller\Module
```
RFY\BKWI?

dimaip commented on Resources/Private/Partials/Module/SubscriberForm.html:
```diff
@@ -0,0 +1,82 @@
+                        <div class="neos-span12">
+                            <div class="neos-control-group neos-row-fluid">
+                                <div class="neos-span10">
+                                    <label class="neos-control-label" for="name">Name</label>
```
Would be nice to use I18n

dimaip commented on Resources/Private/Templates/Module/Subscriber/Edit.html:
```diff
@@ -0,0 +1,11 @@
+<f:layout name="Default" />
+
+<f:section name="Title">
+	<h2>Edit Subscriber {subscriber.name}</h2>
```
Same, I18n

dimaip commented on Resources/Private/Templates/Module/Subscriber/Index.html:
```diff
@@ -0,0 +1,53 @@
+					<tbody>
+					<f:for each="{subscribers}" as="subscriber" key="number">
+						<tr>
+							<td>{number}</td>
```
How is this number useful?

dimaip reviewed (changes requested):
Nice! Left some comments.

svparijs committed:
[TASK] Several changes made to usability

- Removed references to other packages
- Added l18n translations in overall package
- Added filter feature
- Added pagination
